scripts/linters/js_ts_linter.py

```diff
@@ -51,6 +51,17 @@
 TS_IGNORE_EXCEPTIONS = json.load(python_utils.open_file(
     TS_IGNORE_EXCEPTIONS_FILEPATH, 'r'))
 
+INJECTABLES_TO_IGNORE = [
```

Comment:
Explain why they should be ignored -- there doesn't seem to be any pattern to the ignored stuff.

Reply:
Explained.